oat-sa/tao-test-runner-qti-fe
# Feature/tca 558/add h2 to main

**Related to**
https://oat-sa.atlassian.net/browse/TCA-558

**Description**
Add `<h2>` to `<main>` tag and link with Landmark

**How to test**
1. Update taoAct extension
2. As a user start a test
3. Header with the following content should be present in `<main>` tag:
 ```
<h2 id="test-title-header" class="landmark-title-hidden">
      <span data-control="qti-test-item-title">Current item name</span>
      <span data-control="qti-test-item-state">Current item state</span>
</h2>
```
